bin/sapcli

```diff
@@ -86,6 +86,11 @@ def parse_command_line(argv):
                             help="SAP Secure Login Client library (e.g. "
                                  "/Applications/Secure Login Client.app/Contents/MacOS/lib/libsapcrypto.dylib")
 
+    arg_parser.add_argument("--rest-over-rfc", action='store_true', dest="rest_over_rfc",
```

Comment:
Yay, I didn't notice earlier. The generic name 'rest-over-rfc' suggest that all HTTP traffic will be routed via RFC but I do believe only ADT HTTP traffic can be tunneled via RFC. Am I right?

Reply:
That is a very interesting question. In theory, all of the HTTP requests can be tunneled because it directly connects to the HTTP dispatcher on Netweaver Server. However, I did not try with other types of requests and it is not implemented.

Reply:
OK, so since the implementation does not route entire HTTP traffic via RFC, I suggest  to rename to "adt_rest_over_rfc".

Reply:
That is a good point. 
I think it is better to keep the name but show an error message. With this, the parameter would not have to change if this is also implemented for gtcs. This is ultimately your call.